### PR TITLE
Added formal regression for MS MARCO passage task

### PR DESCRIPTION
**Pull request by lintool**
Ref #690 

+ added MS MARCO passage regression into Anserini's framework
+ added tsv topic reader
+ "msmarco" -> "msmarco-passage" to make way for doc task
+ fixed bug with Java timing of MS MACRO retrieve class

### CODE REVIEW

File: docs/experiments-msmarco-passage.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
-# Anserini: Experiments on [MS MARCO](http://www.msmarco.org/)
+# Anserini: Experiments on [MS MARCO (Passage)](https://github.com/microsoft/MSMARCO-Passage-Ranking)
 
 ## Data Prep
 
+We're going to use `msmarco-passage/` as the working directory.
 First, we need to download and extract the MS MARCO dataset:
 
 ```
-DATA_DIR=./msmarco_data
```
Comment (rodrigonogueira4) [inline docs/experiments-msmarco-passage.md]: Much better!

File: docs/regressions-msmarco-passage.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+```
+
+## Effectiveness
+
+With the above commands, you should be able to replicate the following results:
+
+MAP                                     | BM25 (Default)| +RM3      | BM25 (Tuned)| +RM3      |
+:---------------------------------------|-----------|-----------|-----------|-----------|
+[MS MARCO Passage Ranking: Dev Queries](https://github.com/microsoft/MSMARCO-Passage-Ranking)| 0.1924    | 0.1661    | 0.1956    | 0.1766    |
```
Comment (rodrigonogueira4) [inline docs/regressions-msmarco-passage.md]: Amazing!

File: docs/regressions-msmarco-passage.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+nohup target/appassembler/bin/SearchCollection -topicreader Tsv -index lucene-index.msmarco-passage.pos+docvectors+rawdocs -topics src/main/resources/topics-and-qrels/topics.msmarco-passage.dev-subset.txt -output run.msmarco-passage.bm25-tuned.topics.msmarco-passage.dev-subset.txt -bm25 -k1 0.82 -b 0.72 &
+
+nohup target/appassembler/bin/SearchCollection -topicreader Tsv -index lucene-index.msmarco-passage.pos+docvectors+rawdocs -topics src/main/resources/topics-and-qrels/topics.msmarco-passage.dev-subset.txt -output run.msmarco-passage.bm25-tuned+rm3.topics.msmarco-passage.dev-subset.txt -bm25 -k1 0.82 -b 0.72 -rm3 &
+
+```
+
+Evaluation can be performed using `trec_eval`:
+
+```
+eval/trec_eval.9.0.4/trec_eval -m map -m recall.1000 src/main/resources/topics-and-qrels/qrels.msmarco-passage.dev-subset.txt run.msmarco-passage.bm25-default.topics.msmarco-passage.dev-subset.txt
```
Comment (rodrigonogueira4) [inline docs/regressions-msmarco-passage.md]: Was the -c option removed at some point? It ensures that queries that had zero docs returned by Anserini will be included in the calculation of the metrics.

Comment (lintool) [reply docs/regressions-msmarco-passage.md]: I'll add back in

Comment (rodrigonogueira4) [reply docs/regressions-msmarco-passage.md]: Ok, waiting for that to give an approval.

**Pull request merged**